src/monitoring_service/handlers.py

```diff
@@ -383,10 +384,26 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
     user_address = monitor_request.non_closing_signer
     user_deposit = context.user_deposit_contract.functions.effectiveBalance(user_address).call()
 
+    if monitor_request.reward_amount < context.min_service_fee:
+        log.info('Monitor request not executed due to insufficient reward amount',
+                 monitor_request_signer=monitor_request.signer,
```

Comment:
instead of having all fields, you can also just write: `monitor_request=monitor_request`

Reply:
good point and the min_service_fee